opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -573,7 +433,7 @@ message HistogramDataPoint {
 }
 
 // SummaryDataPoint is a single data point in a timeseries that describes the
-// time-varying values of a Summary metric.
+// time-varying values of a Summary metric. It is considered legacy. See comments for Summary
```

Comment:
Please undo this minor addition. This is an opinion that is not universally held within the Prometheus community, and until the OTel histogram options are extended beyond explicit boundaries, the Histogram points are not an alternative to Summary points.

Reply:
@jmacd  done. Just for my understanding then, as `Summary` is the only user of `SummaryDataPoint`, are we saying that there is disagreement about both or are there situations where `SummaryDataPoint` could be used independent of `Summary` in the future?